.gitignore

```diff
@@ -1,5 +1,6 @@
 client/public/bundle.js
 config.js
+.googleConfig.js
```

Comment:
@yulanrong - Instead of creating `.googleConfig.js`, could we just add the `GOOGLE_API` to @EricBaldwinn's existing `config.js`?